### PR TITLE
a more reasonable lower bound for J

### PR DESCRIPTION
**Pull request by bqpd**

### CODE REVIEW

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
test models please

File: gpkitmodels/GP/aircraft/tail/tail_boom.py [hunk 1]
```diff
@@ -161,7 +160,7 @@ def setup(self, tailboom, htail, state):
 
         self.tailboomJ = hasattr(tailboom, "J")
         if self.tailboomJ:
-            constraints.append(tailboom.J >= 1e-50*units("m^4"))
+            constraints.append(tailboom.J >= 1e-10*units("m^4"))
```
Comment (mjburton11) [inline gpkitmodels/GP/aircraft/tail/tail_boom.py]: was `e-50` causing issues?

Comment (bqpd) [reply gpkitmodels/GP/aircraft/tail/tail_boom.py]: It was showing up in Bounded since it's below our default threshold of 1e-30

**Pull request merged**